### PR TITLE
Update Observability installer with Authorization and troubleshooting

### PR DESCRIPTION
**Pull request by shaynafinocchiaro**
# Description
This PR updates Observability installer usage details and adds to Observability troubleshooting.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/112 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/observability/deployment/online.md [hunk 1]
```diff
@@ -121,17 +118,23 @@ To perform an online installation of CSM for Observability, the following steps
...
     |
-    |- Copying Secret from vxflexos to karavi                           Success
+    |- Enabling Karavi Authorization for Karavi Observability
+      |
+      |--> Copying ConfigMap from vxflexos to karavi                    Success
+      |
+      |--> Copying Karavi Authorization Secrets from vxflexos to karavi Success
     |
-    |- Enabling Karavi Authorization for Karavi Observability           Success
+    |- Installing Karavi Observability helm chart                       Success
```
Comment (alikdell) [inline content/docs/observability/deployment/online.md:137]: CSM Observability?

File: content/docs/observability/deployment/online.md [hunk 2]
```diff
@@ -121,17 +118,23 @@ To perform an online installation of CSM for Observability, the following steps
...
+    |- CSI Driver for PowerStore is installed                           Success
     |
-    |- Installing Karavi Observability helm chart                       Success
+    |- Copying Secret from powerstore to karavi                         Success
     |
-    |- Waiting for pods in namespace karavi to be ready                 Success
+    |- Installing CertManager CRDs                                      Success
     |
-    |- Copying Secret from vxflexos to karavi                           Success
+    |- Enabling Karavi Authorization for Karavi Observability
```
Comment (alikdell) [inline content/docs/observability/deployment/online.md:131]: CSM Auth

File: content/docs/observability/deployment/online.md (hunk 1 above)
Comment (shaynafinocchiaro) [reply content/docs/observability/deployment/online.md:137]: These references to Karavi will be addressed at a later time when we do a full name change of our repositories

File: content/docs/observability/deployment/online.md [hunk 3]
```diff
@@ -30,9 +30,8 @@ The Container Storage Modules (CSM) for Observability installer bootstraps Helm
 
 If the Authorization module is enabled for the CSI drivers installed in the same Kubernetes cluster, the installer will perform the current steps to enable CSM for Observability to use the same Authorization instance:
 - Verifies the `karavictl` binary is available.
-- Verifies the appropriate Secret exists in the CSI driver namespace.
-- Queries the CSI driver environment to get references to the Authorization module sidecar-proxy Docker image and URL of the proxy server.
-- Updates the CSM for Observability deployment to use the existing Authorization instance.
+- Verifies the appropriate Secrets and ConfigMap exist in the CSI driver namespace.
+- Enabled during the initial installation of CSM for Observability or updates the CSM for Observability deployment to use the existing Authorization instance.
```
Comment (hoppea2) [inline content/docs/observability/deployment/online.md:34]: Perhaps: "Updates the CSM for Observability deployment to use the existing Authorization instance if not already enabled during the initial installation of CSM for Observability"?

File: content/docs/observability/deployment/online.md [hunk 4]
```diff
@@ -121,17 +118,23 @@ To perform an online installation of CSM for Observability, the following steps
...
     |
+    |- CSI Driver for PowerFlex is installed                            Success
+    |
     |- Copying Secret from vxflexos to karavi                           Success
     |
-    |- Installing CertManager CRDs                                      Success
+    |- CSI Driver for PowerStore is installed                           Success
     |
-    |- Installing Karavi Observability helm chart                       Success
+    |- Copying Secret from powerstore to karavi                         Success
```
Comment (hoppea2) [inline content/docs/observability/deployment/online.md:127]: PowerStore

File: content/docs/observability/deployment/online.md (hunk 3 above)
Comment (shaynafinocchiaro) [reply content/docs/observability/deployment/online.md]: fixed

File: content/docs/observability/deployment/online.md (hunk 4 above)
Comment (shaynafinocchiaro) [reply content/docs/observability/deployment/online.md:127]: The namespace isn't hardcoded in some of the logged steps. The namespace is stored as a variable so the script prints the namespace that's on the cluster here.

**Review by alikdell — APPROVED** [review]

**Review by sharmilarama — APPROVED** [review]

**Pull request merged**